### PR TITLE
Remove Global GParams

### PR DESCRIPTION
**Pull request by antonrohr**
This completely removes the files Params.h and Params.cpp from
OrbitCore. The only value in Params was a global font size and this now
passed as an argument to the classes that need this value.

### CODE REVIEW

**Review by florian-kuebler — APPROVED** [review]
Thanks for cleaning up the code-base. That helps!

**Review by beckerhe — COMMENTED** [review]
I still have some comments! :see_no_evil:

File: OrbitGl/SliderTest.cpp [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
 class MockCanvas : public GlCanvas {
  public:
+  MockCanvas() : GlCanvas(14) {}
```
[the remark this answers is not in the document]
Comment (florian-kuebler) [reply OrbitGl/SliderTest.cpp:13]: Isn't that totally fine for tests?

[the remark this answers is not in the document]
Comment (beckerhe) [reply OrbitGl/SliderTest.cpp:13]: I would say it depends. When you're writing tests that need a lot of numbers (with coordinates or colors for examples), adding constants for all values is cumbersome and does not improve readability.

But in this case I think it helps the reader to grasp what's happening. I think it's not obvious that GlCanvas takes a font size as its argument.

[the remark this answers is not in the document]
Comment (florian-kuebler) [reply OrbitGl/SliderTest.cpp:13]: >I think it's not obvious that GlCanvas takes a font size as its argument.

Depends on your IDE ;). But yes, maybe use a constant here.

**Pull request merged**